newdle/schemas.py

```diff
@@ -131,6 +131,11 @@ class ParticipantSchema(mm.Schema):
         )
     )
 
+    @validates('name')
+    def name_not_empty(self, value):
+        if not value or not value.strip():
```

Comment:
```suggestion
        if not value:
```

We already strip surrounding whitespace [here](https://github.com/indico/newdle/blob/master/newdle/core/webargs.py#L19-L20), so you will never have whitespace-only names